yellowbrick/classifier/rocauc.py

```diff
@@ -124,18 +124,16 @@ class classification is not defined for binary classification problems
 
     Examples
     --------
-    >>> from sklearn.datasets import load_breast_cancer
     >>> from yellowbrick.classifier import ROCAUC
     >>> from sklearn.linear_model import LogisticRegression
     >>> from sklearn.model_selection import train_test_split
-    >>> data = load_breast_cancer()
-    >>> X = data['data']
-    >>> y = data['target']
+    >>> data = load_data("occupancy")
+    >>> features = ["temp", "relative humidity", "light", "C02", "humidity"]
     >>> X_train, X_test, y_train, y_test = train_test_split(X, y)
```

Comment:
In a lot of our documentation, even when we just use the default test size, `test_size=0.25`, we tend to explicitly state it, so we might want to add it here.